### PR TITLE
Add Ios target

### PR DESCRIPTION
**Pull request by Wavesonics**
Adds an iOS meta target,

I'm not sure if there is more to be done on the publishing side? But this compiles for me.

### CODE REVIEW

File: buildSrc/src/main/kotlin/com/akuleshov7/Versions.kt [hunk 1]
```diff
@@ -6,6 +6,6 @@
 object Versions {
     const val KOTLIN = "1.6.21"
     const val JUNIT = "5.7.1"
-    const val OKIO = "3.0.0"
+    const val OKIO = "3.1.0"
```
Comment (Wavesonics) [inline buildSrc/src/main/kotlin/com/akuleshov7/Versions.kt:9]: This added the iOS common target

File: ktoml-core/src/iosMain/kotlin/com/akuleshov7/ktoml/utils/UtilsIos.kt [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+@Suppress("MAGIC_NUMBER")
+internal actual fun StringBuilder.appendCodePointCompat(codePoint: Int): StringBuilder = when (codePoint) {
+    in 0 until Char.MIN_SUPPLEMENTARY_CODE_POINT -> append(codePoint.toChar())
+    in Char.MIN_SUPPLEMENTARY_CODE_POINT..Char.MAX_CODE_POINT -> {
+        append(Char.MIN_HIGH_SURROGATE + ((codePoint - 0x10000) shr 10))
+        append(Char.MIN_LOW_SURROGATE + (codePoint and 0x3ff))
+    }
+    else -> throw IllegalArgumentException()
+}
```
Comment (Wavesonics) [inline ktoml-core/src/iosMain/kotlin/com/akuleshov7/ktoml/utils/UtilsIos.kt:11]: I just copied this code from the linux version, is there anything platform specific needed?

Comment (akuleshov7) [reply ktoml-core/src/iosMain/kotlin/com/akuleshov7/ktoml/utils/UtilsIos.kt:11]: Nope, I do not even think that this method is really a cross-platform code and cannot be in common :)

**Comment by akuleshov7** [comment]
I will make a release after https://github.com/akuleshov7/ktoml/issues/145 I will add all other platforms this week

**Comment by akuleshov7** [comment]
@Wavesonics thank you for your contribution, merged

**Pull request merged**